### DISCUSSION TITLE
Elsa.Webhooks.Extensions namespace missing in library versions newer than 3.0.0

### DISCUSSION THREAD

**Discussion by avacoderhotmail** [elsa-workflows/elsa-core#6237] [Q&A]
I'm trying to run all of the demo apps in the documentation. I have Elsaserver up and running as long as I use Elsa library version 3.0.0. If I try to upgrade to any newer version, the build throws an error that the namespace Elsa.Webhooks.Extensions does not exist. What changes do I need to make in order to use the newer libraries?

**Comment by sfmskywalker** [comment]
From Elsa 3.1 onward, `Elsa.Webhooks.Extensions` was moved to `Elsa.Extensions`. Remove this line:

```csharp
using Elsa.Webhooks.Extensions;
```

and keep/add:

```csharp
using Elsa.Extensions;
```

Your existing `.UseWebhooks(...)` call remains the same. The project made that namespace change before 3.1.0, and the official webhook samples removed the old import in the same update.

Relevant source: [`ModuleExtensions`](https://github.com/elsa-workflows/elsa-core/blob/3.1.0/src/modules/Elsa.Webhooks/Extensions/ModuleExtensions.cs#L1-L19) and the [namespace-change commit](https://github.com/elsa-workflows/elsa-core/commit/64c4d897393c34a1ead1084609f9bbdad3876466).
